reV/supply_curve/supply_curve.py

```diff
@@ -554,43 +862,80 @@ def _calculate_total_lcoe_friction(self):
             logger.info('Found mean LCOE with friction. Adding key '
                         '"total_lcoe_friction" to trans table.')
 
-    def full_sort(self, trans_table=None, sort_on='total_lcoe',
-                  columns=('trans_gid', 'trans_capacity', 'trans_type',
-                           'trans_cap_cost', 'dist_mi', 'lcot', 'total_lcoe')):
+    def _competitive_wind_farms(self, wind_dirs, sc_gid, downwind=False):
         """
-        run supply curve sorting in serial
+        Exclude non-competitive wind farms for given sc_gid
 
         Parameters
         ----------
-        trans_table : pandas.DataFrame | NoneType
+        wind_dirs : CompetitiveWindFarms
+            Pre-initilized CompetitiveWindFarms instance
+        sc_gid : int
+            Supply curve gid to exclude non-competitive wind farms around
+        downwind : bool, optional
+            Flag to remove downwind neighbors as well as upwind neighbors,
+            by default False
+
+        Returns
+        -------
+        wind_dirs : CompetitiveWindFarms
+            updated CompetitiveWindFarms instance
+        """
+        gid = wind_dirs['sc_gid', sc_gid]
```

Comment:
I had a similar comment in the getter: what operation is this performing? Just the the variable names I have absolutely no idea.

Reply:
see new method